Production/govuk_ios/ViewControllers/LaunchViewController.swift

```diff
@@ -2,24 +2,50 @@ import UIKit
 import Foundation
 
 class LaunchViewController: BaseViewController {
-    private lazy var activityIndicator = UIActivityIndicatorView(style: .large)
+    private lazy var animationView = AnimationView.launch
+
+    private let viewModel: LaunchViewModel
+
+    init(viewModel: LaunchViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
         configureUI()
         configureConstraints()
-        activityIndicator.startAnimating()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        animationView.animateIfAvailable(
+            completion: { [weak self] in
+                self?.viewModel.animationCompleted()
+            }
+        )
     }
 
     private func configureUI() {
         view.backgroundColor = .splashScreenBlue
-        activityIndicator.color = .white
-        view.addSubview(activityIndicator)
+        view.addSubview(animationView)
     }
 
     private func configureConstraints() {
-        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
-        activityIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
-        activityIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        animationView.translatesAutoresizingMaskIntoConstraints = false
+        animationView.rightAnchor.constraint(
+            equalTo: view.layoutMarginsGuide.rightAnchor,
+            constant: -20
+        ).isActive = true
+        animationView.leftAnchor.constraint(
+            equalTo: view.layoutMarginsGuide.leftAnchor,
+            constant: 20
```

Comment:
Should these two constant values of `-20` and `20` both use the same constant set somewhere in the class? `animationInset` or something?

Reply:
I have added layout margins to the BaseViewController instead